### ISSUE TITLE
Optimize Images

### ISSUE THREAD

**Issue by poudink** [poudink/Plastik#6]
Most of the PNGs and SVGs probably haven't been optimized well, running an optimizer should save some space.

**Comment by poudink** [comment]
Just finished running `oxipng -o 4 -r -s -Z *` on the whole icon theme. Went from 23.9MB to 21.4MB (or 21.7MB? it gained .3MB when I overwrote the non-optimized version with the optimized copy, not sure from what), saving 2.5MB. Not as much as I was hoping, but this is fine. I plan to take care of optimizing SVGs and most importantly using symlinks to get rid of dupes (#20) very soon. Once all of that is done and combined with #1 having just been completed, I expect CrystalSVG to have slimmed down considerably.

I might also rerun optipng. I think a few files got skipped over due to a couple of errors, not much tho. Optipng takes forever running on the whole icon theme, so this is low priority for me.

**Comment by poudink** [comment]
I decided to take care of the SVGs right now after all. With 4c0dcf6, I used svgcleaner to reduce the size of the "scalable" folder from 3.2MB to 1.9MB, saving 1.3MB. Some of the SVGs had errors, mostly ones from Digikam, which I should probably do something about later.

I haven't touched the Plasma Style yet. Frankly, optimizing it has become the least of my worries, since I've been contemplating redoing most of it to make it more accurate and to make it finally work with color schemes. Once I do get to doing that, I'll make sure it's optimized out of the box. This can now be closed.
